lighthouse-core/gather/connections/cri.js

```diff
@@ -74,10 +75,16 @@ class CriConnection extends Connection {
         });
         response.on('end', _ => {
           if (response.statusCode === 200) {
-            resolve(JSON.parse(data));
-            return;
+            try {
+              resolve(JSON.parse(data));
+              return;
+            } catch (e) {
+              if (data === 'Target is closing') {
```

Comment:
The [chromium code](https://cs.chromium.org/chromium/src/content/browser/devtools/devtools_http_handler.cc?q=%22Target+is+closing%22&sq=package:chromium&l=593&dr=C) that handles this says

```c++
SendJson(connection_id, net::HTTP_OK, NULL, "Target is closing");
```

But unfortunately, it doesn't send JSON. It sends this raw string. 
Thus, the try/catch on this guy.